crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1121,9 +1121,11 @@ impl AuthorityPerEpochStore {
                             })
                         },
                     )),
-                ))
+                )
             } else {
-                None
+                fatal!(
```

Comment:
Can we fully remove this though? What about replaying old tx?

Reply:
agreed with @aschran